packages/config/src/config/vector.style.ts

```diff
@@ -40,6 +40,24 @@ export interface Layer {
   'source-layer'?: string;
 }
 
+/** Sky MapLibre Style Spec, all valuables support interpolate expressions, as unknown format */
+export interface Sky {
+  /** The base color for the sky. Optional Defaults to #88C6FC */
+  'sky-color'?: unknown;
```

Comment:
since we know what these are meant to be could we type them properly? `string` `number` etc?

Reply:
These all support interpolate expressions, we can use `string | unknown` or just `unknown`

Reply:
`string | unknown[]` ?

Reply:
Updated